client/my-sites/marketing/connections/connections.jsx

```diff
@@ -19,7 +20,14 @@ const SharingConnections = ( { translate } ) => (
 		<QueryKeyringConnections />
 		<QueryKeyringServices />
 		<QueryPublicizeConnections selectedSite />
-		<SharingServicesGroup type="publicize" title={ translate( 'Publicize posts' ) } />
+		<SharingServicesGroup
+			type="publicize"
+			title={ translate( 'Publicize posts {{learnMoreLink}}{{/learnMoreLink}}', {
```

Comment:
Is this meant to have a "Learn more" link too? If not, I think it could be re-written like this which could be more consistent to similar scenarios in Calypso:

```suggestion
			title={ translate( 'Publicize posts {{learnMoreLink/}}', {
```